ee/settings.py

```diff
@@ -19,20 +19,9 @@
 HOOK_FINDER = "ee.models.hook.find_and_fire_hook"
 HOOK_DELIVERER = "ee.models.hook.deliver_hook_wrapper"
 
-# Social auth
```

Comment:
These lines (22 - 32) were copied verbatim to L61 - onwards to account for needed variable access at the bottom.